### PR TITLE
Adds docker image for CNI install

### PR DESCRIPTION
**Pull request by trozet**
Travis CI will now run tests and build nimbess-cni docker container
image. The image is uploaded upon merge into master. An install cni
script is added by this patch which allows for k8s to install Nimbess
cni from the docker image. Dependency management also added with
govendor.

Signed-off-by: Tim Rozet <trozet@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,19 @@
...
+
+# jetbrains ide stuff
+.idea*
+
+.swp*
+.swo*
+
+
+bin/
+vendor/github.com/
```
Comment (shague) [inline .gitignore:22]: Don't you want this to be part of the commit? Or does the vendor.json ensure everything gets pulled down?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.11
+WORKDIR $GOPATH/src/github.com/nimbess/nimbess-cni
+COPY . .
+RUN ./build/build-go.sh
+
+FROM centos:latest
```
Comment (shague) [inline Dockerfile:6]: does bess now build with centos?

File: .gitignore (hunk 1 above)
Comment (trozet) [reply .gitignore:22]: vendor.json will pull everything with a "govendor sync" command, so we dont need to store any 3rd party source in our repo

File: Dockerfile (hunk 2 above)
Comment (trozet) [reply Dockerfile:6]: i dont know if bess does or not.

**Review by shague — APPROVED** [review]

**Pull request merged**